src/objective/rank_objective.hpp

```diff
@@ -156,6 +182,7 @@ class LambdarankNDCG : public RankingObjective {
     std::stable_sort(
         sorted_idx.begin(), sorted_idx.end(),
         [score](data_size_t a, data_size_t b) { return score[a] > score[b]; });
+
```

Comment:
Can you please remove these and other whitespace-only changes?

![image](https://user-images.githubusercontent.com/7608904/130723144-cae95a13-9a8d-4cfd-aadf-0cd9a4cf115a.png)

If you feel that they improve the readability of the code, we'd welcome a separate pull request proposing them. That separate pull request could also include some of the other changes you've made below where you added or updated comments on existing code (where those comment changes don't relate to the feature of "add position-unbiased option for lambdarank").

like these:

![image](https://user-images.githubusercontent.com/7608904/130723495-cd519662-46df-44e9-89a2-2bab356bd7af.png)

![image](https://user-images.githubusercontent.com/7608904/130723511-7d002483-4cfe-4510-92d9-0e97eced477e.png)

----

Those types of PRs that only change comments and whitespace take minimal effort for reviewers to review, and shrink the size of changes like this one that are introducing more involved enhancements.

Reply:
i think i undid all the whitespace changes